From cbd5e6160fe0561981a4bc5dcfbd07121e0382d9 Mon Sep 17 00:00:00 2001
From: Muhammad Husaini Zulkifli <muhammad.husaini.zulkifli@intel.com>
Date: Tue, 16 Mar 2021 02:50:40 +0800
Subject: [PATCH 083/122] igc: Revert back xdp_rxq_info_reg() registration call

Modified not passing the napi_id parameter to xdp_rxq_info_reg()
in v5.10 kernel as this only available in v5.11 Kernel onwards.

Signed-off-by: Muhammad Husaini Zulkifli <muhammad.husaini.zulkifli@intel.com>
---
 drivers/net/ethernet/intel/igc/igc_main.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/intel/igc/igc_main.c b/drivers/net/ethernet/intel/igc/igc_main.c
index 30c1625ec52d..110e245fd593 100644
--- a/drivers/net/ethernet/intel/igc/igc_main.c
+++ b/drivers/net/ethernet/intel/igc/igc_main.c
@@ -495,8 +495,7 @@ int igc_setup_rx_resources(struct igc_ring *rx_ring)
 	u8 index = rx_ring->queue_index;
 	int size, desc_len, res;
 
-	res = xdp_rxq_info_reg(&rx_ring->xdp_rxq, ndev, index,
-			       rx_ring->q_vector->napi.napi_id);
+	res = xdp_rxq_info_reg(&rx_ring->xdp_rxq, ndev, index);
 	if (res < 0) {
 		netdev_err(ndev, "Failed to register xdp_rxq index %u\n",
 			   index);
-- 
2.17.1

